### ISSUE TITLE
Incorrect color conversion?

### ISSUE THREAD

**Issue by themightyoarfish** [murtiad/las2pcd#6]
https://github.com/murtiad/las2pcd/blob/2d6707a0df367e35b7006d8a955f394e8b42e068/las2pcd.cpp#L78

The code multiplies by 256 here, should it not be 255?

**Comment by murtiad** [comment]
Hi @themightyoarfish. CMIIW but for the conversion I used 256 instead of 255 because there are 256 available grey shades (0-255)

**Comment by themightyoarfish** [comment]
Yes, but the highest value is 255. So if the expression results in something between 0 and 1.0, we have to multiply by the largest value, no?
Maybe I'm misunderstanding something but it looks as if this code could generate color values of 256 which would be invalid for 8 bit rgb.

**Comment by murtiad** [comment]
I see your point and I think you're right. If we have a max las RGB value of 65535 this would indeed cause the code to generate a value of 256 in 8 bit which would be incorrect. I'll modify the code accordingly, thanks a lot for pointing out this important detail!
